modules/templates/util_render.go

```diff
@@ -224,3 +224,25 @@ func RenderLabels(ctx context.Context, labels []*issues_model.Label, repoLink st
 	htmlCode += "</span>"
 	return template.HTML(htmlCode)
 }
+
+func RenderDependencies(ctx context.Context, dependencies []*issues_model.DependencyInfo) template.HTML {
+	if len(dependencies) == 0 {
+		return ""
+	}
+
+	htmlCode := "<span>"
+
+	for index, dependency := range dependencies {
+		if index != 0 {
+			htmlCode += `<span>,</span>`
+		}
+		anchorClasses := "gt-ml-2"
+		if dependency.Issue.IsClosed {
+			anchorClasses += " term-fg9"
```

Comment:
`term-fg9` is not a general color for frontend. IIRC it is only for the "terminal rendering"

There should be other existing color helper classes.

Reply:
Its not a color, but I understand what you mean